nuclia/sdk/search.py

```diff
@@ -192,23 +192,37 @@ def ask_json(
         See https://docs.nuclia.dev/docs/api#tag/Search/operation/Ask_Knowledge_Box_kb__kbid__ask_post
         """
         ndb: NucliaDBClient = kwargs["ndb"]
+        if isinstance(schema, str):
+            if os.path.exists(schema):
+                with open(schema, "r") as json_file_handler:
+                    schema_json = json.load(json_file_handler)
```

Comment:
should we try/except here and return a proper error log if the contents of the file could not be loaded?